README.md

```diff
@@ -260,36 +177,123 @@ quantized_model = nncf.quantize(model, calibration_dataset)
 
 </details>
 
+
+[//]: # (NNCF provides full  [samples]&#40;#post-training-quantization-samples&#41;, which demonstrate Post-Training Quantization usage for PyTorch, TensorFlow, ONNX, OpenVINO.)
+
+### Training-time Compression
+
+The basic workflow steps:
+1) Create an NNCF config object from (a) JSON configuration file and (b) dataloader for the compression initialization.
+2) Pass your model together with the config to the `create_compressed_model` function which returns two entities:
+   1) Compression controller — an object allowing you to control the compression, for example during training.
+   2) Compressed model which may further be used as a regular PyTorch model for fine-tuning and inference.
+
+Expand the blocks below for PyTorch and Tensorflow usage examples: 
+
+<details><summary><b>PyTorch</b></summary>
```

Comment:
Added expanding code blocks for QAT too. They are rather large, hidden by default

Reply:
What you think about move examples of using to QuickStart.md

In main readme leave only a short example for OV backend, as more interesting for users.

Reply:
I haven't found a `QuickStart.md` you are referring to. Did you imply to create it? Honestly, I think it's ok to keep all code examples as they're hidden anyway.

But it's worth considering moving QAT usage section from readme to an improved `Usage.md` because it's kinda heavy for readme in my opinion. But that's in the future, I'd like to move in small incremental steps.